src/api/app/services/scm_exception_handler.rb

```diff
@@ -51,6 +52,6 @@ def initialize(event_payload, event_subscription_payload, scm_token, workflow_ru
   private
 
   def log_to_workflow_run(exception, scm)
-    @workflow_run.update_to_fail("Failed to report back to #{scm}: #{exception.message}")
+    @workflow_run.update_to_fail("Failed to report back to #{scm}: #{exception_message(exception: exception, scm: scm)}")
```

Comment:
Instead of including the Module, you can rewrite it as a class and ask for something like `ScmExceptionMessages.for(scm).explanation(exception)`.

Reply:
that sounds good. I'll update it.